frontend/office.js

```diff
@@ -119,7 +119,18 @@ $(() => {
       width: "100%",
       height: "100%",
       parentNode: document.querySelector("#meet"),
+      configOverwrite: {
+        //startAudioOnly:true,
+        resolution :360,
```

Comment:
lint it 💄

Reply:
could you update my branch with this lint? :)